qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -54,7 +55,7 @@ class FermionicOp(ParticleOp):
     There are two types of label modes for :class:`FermionicOp`.
     The label mode is automatically detected.
 
-    1. Dense Label (if underscore `_` does not exist in the label)
+    1. Dense Label (default, `register_length = None`)
 
     Dense labels are strings with allowed characters above.
     This is similar to Qiskit's string-based representation of qubit operators.
```

Comment:
In this case register_length is inferred from the label length. I guess its a mistake to state it explicitly, when they are dense labels, as its not going to work?